tasks/linux.yml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
I wonder how feasible it is to merge the `linux.yml` and `windows.yml` files. The main difference seems to be the file paths and certain commands. Did we decide to split them because they became overly complicated as one file?

Reply:
The windows playbook invokes different modules (prefixed by win_) which take different parameters (ex. mode) than the linux playbook and i'm not aware of a way to conditionally add the prefix to the module name or conditionally add certain parameters.

Also since the logging agent for windows integration wont follow the same pattern of calling the script we'd likely need an additional playbook regardless.

Reply:
Got it.